M2/Macaulay2/m2/typicalvalues.m2

```diff
@@ -199,6 +201,8 @@ generateTypicalValues = (srcdir) -> (
 
 if isMember("--no-tvalues", commandLine) then end
 
+if not fileExists typicalValuesSource then generateTypicalValues(currentFileDirectory | "../d/")
```

Comment:
I don't know how likely this situation would be, but suppose that `tvalues.m2` doesn't exist and we're reading this file from someplace other than the source repository so that `../d` doesn't exist.  Should the user get a more informative error message that the one they'll get from `readDirectory`?

Reply:
I think this is the error you'd get:
```m2
../../Macaulay2/m2/debugging.m2:18:6:(1):[12]: error: unable to find the source code for the interpreter
```
(not sure why it says it's coming from `debugging.m2` ... I find this so annoying.)

Reply:
Oh, I missed that check! It was added it a commit I hadn't looked at yet

Reply:
No worries. Also, if you or @pzinn have any clues about why the error points to debugging.m2 rather than the real source, I'd be very curious.

Reply:
No clue, but this has always bugged me too. Sometimes I've found that if run the code that raised the error a second or third time, I'll eventually get a useful backtrace with the actual source of the error. It's very weird!

Reply:
Maybe once this PR is merged I'll make an issue with this isolated example which seems to be pretty reproducible.